resources/js/alpine/newsCarouselComponent/newsCarouselComponent.test.ts

```diff
@@ -10,6 +9,7 @@ import {
   it,
   vi,
 } from 'vitest';
+import Alpine from '../../../../vendor/livewire/livewire/dist/livewire';
```

Comment:
@wescopeland not sure how to deal with the failed test. Alpine doesn't seem to be happy how we're using it but i couldn't figure out how to overcome this.
Maybe you can help finding what the issue is? 🙏

Reply:
I think the correct approach to solving this is to set up an alias in the vite config, eg:

```ts
resolve: {
  alias: {
    '@': resolve(__dirname, './resources/js'),
    alpine: resolve(__dirname, './vendor/livewire/livewire/dist/livewire.js')
  },
},
```

Unfortunately I'm not having much luck with this, even though it does appear the file is trying to export Alpine:

```js
// cjs (livewire.js)
window.Alpine = module_default;

// esm (livewire.esm.js)
var export_Alpine = import_alpinejs20.default;
export {
  export_Alpine as Alpine,
  Livewire2 as Livewire
};
```

I can get as far as the tests trying to run, but then some kind of DOM error is output from an Alpine.js internal and I can't get much further.

I think we have two options:
* Install alpine as a dev dependency for the sole purpose of testing.
* Delete this test suite.

I have no strong feelings or opinion about either approach.

Reply:
assuming we'd aim for replacing all of the custom alpine components and only use livewire directly in the future we might be good to remove the tests?

Reply:
maybe too soon - went with the dev dependency instead

Reply:
sounds good :) thank you for looking into this!

Reply:
Sure thing!